.github/workflows/ubuntu.yml

```diff
@@ -54,12 +65,23 @@ jobs:
           set -x
           cd ${LOCAL_BUILDS}/rnp-build
           cpack -G DEB -D CPACK_DEBIAN_PACKAGE_SHLIBDEPS_PRIVATE_DIRS="${BOTAN_INSTALL}/lib;${JSONC_INSTALL}/lib;${GPG_INSTALL}/lib"
+          
   cmake-offline-googletest-src:
+    strategy:
+      matrix:
+        cc: ["gcc"]
+        cxx: ["g++"]
+        include:
+          - cc: clang
```

Comment:
This test is just to check that we build correctly with Googletest from sources via CMake define, no need to check with different compilers.

Reply:
Same as above - it really tests ci scripts

Reply:
These tests are commented